qutebrowser/browser/adblock.py

```diff
@@ -111,9 +111,9 @@ def __init__(self):
 
         config.instance.changed.connect(self._update_files)
 
-    def is_blocked(self, url):
+    def is_blocked(self, url, primary=None):
         """Check if the given URL (as QUrl) is blocked."""
-        if not config.val.content.host_blocking.enabled:
+        if not config.instance.get("content.host_blocking.enabled", url=primary):
```

Comment:
Nitpick: Please use single quotes here (I try to use single quotes for "keys" like this, and double-quotes for "actual" strings shown to the user).

Reply:
You'll also need to split this into two lines (e.g. after the comma)